src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(int radius, String color) {
+        super(String.valueOf(color));
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: circle, area: "
+                + String.format("%.1f", getArea())
+                + " sq. units, radius: " + radius + " units, color: " + getColor());
```

Comment:
Instead of concatenating strings in the 'draw' method, consider using a single String.format for better readability.